prow/gerrit/fakegerrit/fakegerrit.go

```diff
@@ -80,6 +86,23 @@ func (fg *FakeGerrit) GetComments(id string) map[string][]*gerrit.CommentInfo {
 	return nil
 }
 
+func (fg *FakeGerrit) AddChangeAfter(projectName string, change *gerrit.ChangeInfo) {
```

Comment:
this function seems very much like a client side function instead of a server side function, other than the fact that it relies on a monotonically increasing `lastUpdateTime` from the fake server, which I think could be done from client side as well, what do you think?

Reply:
imo each integration test should operate independent of each other, for example:
- `TestA` should operate on `orgA`
- `TestB` should operate on `orgB`

It would be a bug if `lastUpdateTime` of `orgA` has any effect on `orgB`. So a clock in each integration test should be good enough?